x/smart/README.md

```diff
@@ -0,0 +1,132 @@
+# Smart Dialer
+
+The **Smart Dialer** searches for a strategy that unblocks DNS and TLS for a given list of test domains. It takes a config describing multiple strategies to pick from.
+
+## JSON config for the Smart Dialer
+
+The config that the Smart Dialer takes is in a JSON format. Here is an example:
+
+```json
+{
+  "dns": [
+    {
+      "https": {
+        "name": "doh.sb"
+      }
+    }
+  ],
+  "tls": [
+    "override:host=cloudflare.net|tlsfrag:1"
+  ]
+}
+
```

Comment:
I think it would be clearer if use use multiple options. Also, we should have no circumvention as the first option. I'd I'd rather avoid `dns.sb` which is more obscure.

```suggestion
{
  "dns": [
      {"system": {}},
      {"https": {"name": "8.8.8.8"}},
      {"https": {"name": "9.9.9.9"}}
  ],
  "tls": [
      "",
      "split:2",
      "tlsfrag:1"
  ]
}
```